siriuspy/siriuspy/pwrsupply/pructrl/pru.py

```diff
@@ -71,12 +60,8 @@ def _close(self):
 class PRU(PRUInterface):
     """Functions for the programmable real-time unit."""
 
-    def __init__(self, bbbname=None, ip_address=None):
+    def __init__(self, ethbridgeclient, bbbname=None, ip_address=None):
```

Comment:
since this is not an instance of class but the class itself, I think it would be more appropriate to change the name of this variable to something like this: `ethbriggeclt_class` or `comm_class`

Reply:
I agree. I though about it but was afraid the name would get too long. But it is better long than misleading...

Reply:
done.